src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double loadFactor = 0.75;
+    private int size;
+    private int initialCapacity = 16;
+    private Node<K, V>[] nodeArray;
+
+    public MyHashMap() {
+        nodeArray = new Node[initialCapacity];
+    }
+
+    class Node<K, V> {
+        private int hash;
```

Comment:
In JDK implementation this parameter holds hash code of key(not backet index). I don't think we need this parameter in this task